arviz/tests/helpers.py

```diff
@@ -39,7 +39,6 @@ def chains():
 
 def create_model(seed=10):

```

Comment:
the error in the logs is from this line: `seed` is unused. i agree with @OriolAbril that for now we should 

- leave the seed as an argument (this requires no action)
- add back the two calls to `np.random.seed` that were removed

Reply:
Only one call to `np.random.seed(seed)` was removed and has to be added back though. The one that was in line 42 of `helpers.py`, right after the docstring.